tripper/dataset/tabledoc.py

```diff
@@ -0,0 +1,68 @@
+"""Basic interface for tabular documentation of datasets."""
+
+from typing import TYPE_CHECKING
+
+from tripper import Triplestore
+from tripper.dataset.dataset import addnested, as_jsonld, save_dict
+from tripper.utils import AttrDict
+
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import List, Optional, Sequence, Union
+
+
+class TableDoc:
+    """Representation of tabular documentation of datasets.
+
+    Arguments:
+        header: Sequence of column header labels.  Nested data can
+            be represented by dot-separated label strings (e.g.
+            "distribution.downloadURL")
+        data: Sequence of rows of data. Each row documents an entry.
+        type: Type of data to save (applies to all rows).  Should
+            either be one of the pre-defined names: "dataset",
+            "distribution", "accessService", "parser" and "generator"
+            or an IRI to a class in an ontology.  Defaults to
+            "dataset".
+        prefixes: Dict with prefixes in addition to those included in the
+            JSON-LD context.  Should map namespace prefixes to IRIs.
+        context: Dict with user-defined JSON-LD context.
+
+    """
+
+    # pylint: disable=redefined-builtin,too-few-public-methods
+
+    def __init__(
+        self,
+        header: "Sequence[str]",
+        data: "Sequence[Sequence[str]]",
+        type: "Optional[str]" = "dataset",
+        prefixes: "Optional[dict]" = None,
+        context: "Optional[Union[dict, list]]" = None,
+    ):
+        self.header = header
+        self.data = data
+        self.type = type
+        self.prefixes = prefixes
+        self.context = context
+
+    def asdicts(self) -> "List[dict]":
+        """Return the table as a list of dicts."""
+        kw = {"_context": self.context} if self.context else {}
```

Comment:
Should this be @context?
```suggestion
        kw = {"@context": self.context} if self.context else {}
```

Reply:
> Should this be @context?

Either should work. Maybe "@context" is more readable. We support "_context" since "@context" cannot be provided as a keyword argument (unless using `**` unpacking, like we are doing here).